src/main/common/filter.h

```diff
@@ -75,6 +75,12 @@ typedef enum {
     FILTER_BPF,
 } biquadFilterType_e;
 
+typedef enum {
+    EXP_FILTER_NONE = 0,
```

Comment:
I suggest changing these to `STAGE2_FILTER_NONE`, `STAGE2_FILTER_BIQUAD_RC_FIR2`, `STAGE2_FILTER_FKF`